storage/storage.js

```diff
@@ -42,13 +42,14 @@ class Storage {
 	 * @return Promise
 	 */
 	bootstrap() {
-		const adapter = new PgpAdapter(
-			Object.assign({}, this.options, {
+		const adapter = new PgpAdapter({
+			...this.options,
+			...{
```

Comment:
Here you can directly set the keys without using the spread operator

Reply:
Need to merge this with `this.options` so why have to use spread here.